content/02.introduction.md

```diff
@@ -147,6 +147,20 @@ After a virus enters into a host cell, its antigen is presented by major histoco
 One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
 In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
 
+##### Cytokine Storm
+
+<!-- As I am cleaning up the text, I'll move the cytokine storm text here, as right now it's very decentralized and one of our goals is to merge everything together in the intro!-->
+IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
+In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
+In this way interleukins regulate both the pro- and anti-inflammatory responses.
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
+The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
+However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
```

Comment:
Just to clarify, is the point here that SARS patients have intermediate levels of IL-6 but other respiratory conditions, like ARDS have higher levels of IL-6?

As a non-immunologist, my high level understanding of what is written is: IL-6 plays a role in regulating the immune response to infection. And different IL-6 levels are found depending on the disease. This IL-6 cytokine causes a large immune response that damages cells and is fatal.